build.gradle

```diff
@@ -68,7 +68,7 @@ dependencies {
     implementation("io.micronaut.aws:micronaut-aws-parameter-store")
     implementation "software.amazon.awssdk:ecr"
     implementation "software.amazon.awssdk:ecrpublic"
-    implementation 'com.amazonaws:aws-java-sdk-ses:1.12.471'
+    implementation 'software.amazon.awssdk:ses'
     // include sts to allow the use of service account role - https://stackoverflow.com/a/73306570
     implementation 'com.amazonaws:aws-java-sdk-sts:1.12.471'
```

Comment:
also `aws-java-sdk-sts` needs to be updated

Reply:
we do have this entry of sts but only for run time
https://github.com/seqeralabs/wave/blob/f17be6b21b674790620202cee0232b92d71ddee6/build.gradle#L79

So I will change this to implementation

Reply:
Didn't realised about this. In principle `runtimeOnly` should be enough, because it's not used in the implementation